products-service/functions/catalogBatchProcess.ts

```diff
@@ -0,0 +1,40 @@
+import { SQSHandler } from "aws-lambda";
+import { Client } from "pg";
+import { dbOptions } from "../config";
+
+export const catalogBatchProcess = (sns): SQSHandler => {
```

Comment:
I'm not using typescript in this course, but is it OK using SQSHandler return type in a handler for SNS?